index.html

```diff
@@ -4,9 +4,39 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <meta name="description" content="Login to Megaphone to announce your life">
+    <link rel="stylesheet" href="dist/css/styles.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+    <title>Login</title>
 </head>
 <body>
-    <p>Hello!</p>
+  <img src="./img/megaphone-oleg-laptev-QRKJwE6yfJo-unsplash.jpg" class="position-fixed top-50 start-50 translate-middle" alt="An orange megaphone">
+    <main class="justify-content-center align-items-center mw-100"> 
+        <section class="container">
+          <div class="row align-items-center p-0 position-absolute top-50 start-50 translate-middle w-75">
+            <h1 class="text-center text-md-start h1 mb-5 fw-bold text-capitalized col-sm-12 col-md-8 position-relative">Megaphone Social</h1>
```

Comment:
Same here, child content of a row should always be a col type class

Reply:
Fixed